src/screens/ConstructionSiteDetailScreen.js

```diff
@@ -58,16 +60,28 @@ export const ConstructionSiteDetailScreen = ({ route }) => {
     restrictions,
     startDate,
     title
-  } = constructionSites[index];
-
-  useMatomoTrackScreenView(`${MATOMO_TRACKING.SCREEN_VIEW.CONSTRUCTION_SITE_DETAIL} / ${title}`);
+  } = constructionSites[0];
 
   const extendedTitle = (category ? `${category} - ` : '') + title;
   const formattedDates = formatDates(startDate, endDate);
 
+  if (loading && !refreshing) {
+    return <LoadingSpinner loading />;
+  }
```

Comment:
i am not seeing the spinner when visiting detail screens. can you see it?

Reply:
for me the loading is generally already finished, by the time it has properly transitioned. if we set `true` as the condition here, it properly shows the loading spinner.

did you encounter an empty screen, instead of a loadingspinner at some point, or did you also have the data present immediately?

Reply:
i did some checks and also had the data on screen directly. the state of visible loading spinner did never occur, which is fine i would say.

why is there `!refreshing` included in the condition? we have just `loading` in other places. isn't that enough here as well?

Reply:
you are right, we do not need the `!refreshing` condition here. i will remove it